src/main/java/org/interledger/cryptoconditions/Fulfillment.java

```diff
@@ -24,6 +26,15 @@
 	 * @return the type of the condition that this fulfills
 	 */
 	ConditionType getType();
+	
+	/**
+	 * Get the features requited for this fulfillment
+	 * 
+	 * @see FeatureSuite
+	 * 
+	 * @return the type of the condition that this fulfills
+	 */
+	EnumSet<FeatureSuite> getFeatures();
```

Comment:
Why did you put this on the fulfillment? I guess it saves having to generate the condition to get it. Good thinking!